src/styles/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  max-width: 1200px;
+  display: flex;
+
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  position: relative;
+  padding: 0 50px;
+  margin: 0 auto;
+
+  &__logo {
+    display: block;
+    height: 40px;
+  }
+
+  &__nav {
+    &-list {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      text-align: center;
+      padding: 0;
+
+      &-item {
+        list-style: none;
+        line-height: 60px;
+
+        &:not(:last-child) {
+          margin-right: 20px;
+        }
+
+        &-link {
+          display: block;
+          height: 60px;
+          text-decoration: none;
+          color: #060b35;
+          text-transform: uppercase;
+          position: relative;
+
+          &:hover {
+            color: #00acdc;
+          }
+        }
+      }
+    }
+  }
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
Use a created variable for this color, fix it everywhere 
<img width="221" alt="image" src="https://github.com/user-attachments/assets/505326c5-e098-4c42-93d4-7a893d7256b4">